tests/src/test/java/com/powsybl/openrao/tests/steps/SearchTreeRaoSteps.java

```
@@ -65,6 +65,19 @@ public class SearchTreeRaoSteps {
     private LoopFlowResult loopFlowResult;
     private Network network;
     private State preventiveState;
+    private String raoImplementation;
+
+    {
+        try {
+            final Properties configProperties = new Properties();
+            final InputStream inputStream = this.getClass().getClassLoader().getResourceAsStream("steps-config.properties");
+            configProperties.load(inputStream);
+            raoImplementation = (String) configProperties.get("rao-implementation");
+        } catch (final Exception e) {
+            System.out.println("Could not load configuration file");
```

A little bit weird because we don't usually use this to print message maybe just throw an OpenRaoException and print the stackTrace.

---

Ok, then throw an OpenRaoException instead of printing and going forward. I'll change it that way.